rootfs/api/models/app.py

```diff
@@ -503,15 +503,9 @@ def verify_application_health(self, **kwargs):
         # Get the router host and append healthcheck path
         url = 'http://{}:{}'.format(settings.ROUTER_HOST, settings.ROUTER_PORT)
 
-        # if a health check url is available then 200 is the only acceptable status code
```

Comment:
Any reason for this removal? I specifically added that so we'd it the application as well in case of healthchecks being set